frontend/src/components/positions-list.js

```diff
@@ -3,15 +3,15 @@ import PropTypes from "prop-types";
 
 export class PositionsList extends React.Component {
     static propTypes = {
-        positions: PropTypes.arrayOf(
+        data: PropTypes.arrayOf(
             PropTypes.shape({
                 position_code: PropTypes.string,
                 position_title: PropTypes.string
             })
```

Comment:
I think this is overly restrictive.

Reply:
We need to validate that the positions have the relevant fields before they are rendered in the table, do you think this is not the right place to do this validation?  Should I change it to `data: PropTypes.arrayOf( PropTypes.any)`?

Reply:
I guess this is fine here. I mistook `shape` for `exact`.